docs/guides/migrating-from-camunda-7/data-migrator/cockpit-plugin.md

```diff
@@ -0,0 +1,86 @@
+---
+id: cockpit-plugin
+title: Cockpit plugin
+sidebar_label: Cockpit plugin
+description: "Web-based interface for viewing information about skipped entities during the migration process."
+---
+
+:::warning Experimental feature
+
+- The Cockpit plugin is an **experimental** feature and we don't recommend using it in production environments.
+- You can read more about limitations in the [limitations](/guides/migrating-from-camunda-7/data-migrator/limitations.md#cockpit-plugin) page.
+  :::
+
+The Cockpit plugin provides a web-based interface for viewing information about skipped and migrated runtime and history data. It integrates with Camunda 7 Cockpit to give you visibility into which runtime process instances or history data (like variables, flow nodes, etc.) were successfully migrated or skipped during migration and the reasons why.
+
+For more information on Camunda 7 plugins, see the [Camunda 7 documentation](https://docs.camunda.org/manual/latest/webapps/cockpit/extend/plugins/).
+
+## Prerequisites
+
+:::warning Database Configuration Requirement
+The Cockpit plugin **only works** when the Data Migrator creates the migration schema in the Camunda 7 database (default behavior), or when both Camunda 7 and Camunda 8 databases point to the same database instance.
+
+Since the Cockpit plugin runs within Camunda 7, it can only access databases that Camunda 7 is connected to. For an example, see the [Data Migrator setup example](config-examples.md#data-migrator).
+:::
```

Comment:
Looking at the end result, two prerequisite warnings seem to be a bit much. Also, do these database location requirements need so much attention? Can we add this to the list of prerequisites without a banner? 🤔 

<img width="1011" height="734" alt="Image" src="https://github.com/user-attachments/assets/52a61917-ff6e-4b15-9348-e9be77a52728" />